kv/kvnemesis: TestKVNemesisSingleNode failed

[(kv/kvnemesis).TestKVNemesisSingleNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2558142&tab=buildLog) on [release-20.2@7274fe0d3c5ce963d2f50920670d7e6e4a2969de](https://github.com/cockroachdb/cockroach/commits/7274fe0d3c5ce963d2f50920670d7e6e4a2969de):

Fatal error:
```
panic: test timed out after 45m0s

```

Stack:
```
goroutine 120039 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1377 +0x11c
created by time.goFunc
	/usr/local/go/src/time/sleep.go:168 +0x52

```

<details><summary>Log preceding fatal error</summary><p>

```
=== RUN   TestKVNemesisSingleNode

```

</p></details>

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestKVNemesisSingleNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisSingleNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/kv/kvnemesis/BUILD.bazel
+++ b/pkg/kv/kvnemesis/BUILD.bazel
@@ -62,6 +62,7 @@ go_test(
         "//pkg/server",
         "//pkg/storage",
         "//pkg/testutils/serverutils",
+        "//pkg/testutils/skip",
         "//pkg/testutils/sqlutils",
         "//pkg/testutils/testcluster",
         "//pkg/util/hlc",

--- a/pkg/kv/kvnemesis/kvnemesis_test.go
+++ b/pkg/kv/kvnemesis/kvnemesis_test.go
@@ -19,6 +19,7 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/base"
 	"github.com/cockroachdb/cockroach/pkg/kv"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/testcluster"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
@@ -29,6 +30,8 @@ import (
 
 func TestKVNemesisSingleNode(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	skip.UnderRace(t)
+
 	defer log.Scope(t).Close(t)
 
 	ctx := context.Background()
@@ -52,6 +55,8 @@ func TestKVNemesisSingleNode(t *testing.T) {
 
 func TestKVNemesisMultiNode(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	skip.UnderRace(t)
+
 	defer log.Scope(t).Close(t)
 
 	// 4 nodes so we have somewhere to move 3x replicated ranges to.

